### DISCUSSION TITLE
File open dialog now only lets you open folders but not files

### DISCUSSION THREAD

**Discussion by fumakotaro** [mifi/lossless-cut#1995] [General]
The file open dialog now only displays folders and not video files.

Furthermore I still would love to be able to choose the name of the created output file easily.

**Comment by mifi** [comment]
hi. can you provide some more info? which version? operating system? screenshot?

**Reply by fumakotaro** [reply]
Oh sorry I thought it was an obvious oversight. I am on Windows 10 and downloaded LosslessCut-win-x64.7z.

![image](https://github.com/mifi/lossless-cut/assets/7364951/84c1bd39-8c34-4dfa-9be9-55c3742c0001)

There are tons of .mp4 files in that folder.

**Comment by mifi** [comment]
so it works with the previous version?

**Reply by fumakotaro** [reply]
Yep:

![image](https://github.com/mifi/lossless-cut/assets/7364951/a79ced03-fccd-48c8-8f29-019cb22e00ad)

**Reply by mifi** [reply]
ok, seems to be a problem on Windows only. need to research a bit more

**Reply by mifi** [reply]
I just found https://www.electronjs.org/docs/latest/api/dialog

> Note: On Windows and Linux an open dialog can not be both a file selector and a directory selector, so if you set properties to ['openFile', 'openDirectory'] on these platforms, a directory selector will be shown.

The bug was introduced here: bc04a4f2a4b08ff593c77079cb321a0142330edb

**Reply by fumakotaro** [reply]
Thank you for investigating and finding a fix (I saw your commit)!

**Comment by outlyer-net** [comment]
> ok, seems to be a problem on Windows only. need to research a bit more

As pointed out on the Electron's page you quoted, I can confirm it also affects Linux

> I just found https://www.electronjs.org/docs/latest/api/dialog
> 
> > Note: On Windows and Linux an open dialog can not be both a file selector and a directory selector, so if you set properties to ['openFile', 'openDirectory'] on these platforms, a directory selector will be shown.
